hanimandl.ino

```diff
@@ -178,6 +178,7 @@ const int outputSW = 26;
 
 // Servo
 const int servo_pin = 33;
+// const int servo_pin = 45; // Move Servo to different PIN for HELTEC V3.1
```

Comment:
I also did some tests for the servo - as I did not want to change the PIN - also tried https://github.com/hiveeyes/hanimandl/pull/69 - but that did not do the trick for me.